crates/bevy_animation/src/lib.rs

```diff
@@ -153,6 +155,12 @@ pub struct AnimationClip {
     duration: f32,
 }
 
+#[derive(Clone, Component)]
```

Comment:
Also document the fact that (I think) it's also used as a marker for which entity path correctly resolved from an animation, right? Because I see we're removing the component sometimes, so it's not just a cache, but also a marker?

Reply:
`AnimationTarget` isn't actually used as a marker for successful resolution, because it's independent of the `AnimationClip` playing. This is by design, so that the animation clip can change without having to traverse the subtree and update the `AnimationTarget`s. The only time we remove `AnimationTarget` is when no animation clip could possibly ever target the entity, either because it becomes unnamed or when tree changes make the entity no longer a descendant of any `AnimationPlayer`.